### PR TITLE
Demo, converting views to class-style

### PR DESCRIPTION
**Pull request by dotpointer**
Demo, converting views to class-style.

Testing: `yarn install; yarn start`, go to `http://127.0.0.1:<your port>/components/cosmoz-grouped-list/demo/` where `<your port>` can be `8000` or something else.

Build errors are due to the Chrome 74-76 error.

Issue is https://github.com/Neovici/cosmoz-frontend/issues/815.

### CODE REVIEW

File: demo/helpers/demo-basic.html [hunk 1]
```diff
@@ -62,48 +62,50 @@ <h3>cosmoz-grouped-list demo</h3>
...
+		class DemoBasic extends Polymer.Element {
+			static get is() {
+				return 'demo-basic';
+			}
+			static get properties() {
+				return {
+					data: {
+						type: Array,
+						notify: true,
+						value: function () {
```
Comment (Wurper) [inline demo/helpers/demo-basic.html]: nitpicking - replace `function` for short hand?

File: demo/helpers/demo-list-helper.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<script>
+	window.Cosmoz = window.Cosmoz || {};
+
+	Cosmoz.Demo = Cosmoz.Demo || {};
+
+	function getRandomInt(min, max) {
+		return Math.floor(Math.random() * (max - min + 1)) + min;
+	}
+
+	Cosmoz.Demo.generateListDemoData = function (groupsMin, groupsMax, itemsMin, itemsMax) {
```
Comment (Wurper) [inline demo/helpers/demo-list-helper.html]: nitpicking - replace `function` for short hand?

File: demo/helpers/demo-basic.html (hunk 1 above)
Comment (dotpointer) [reply demo/helpers/demo-basic.html]: Replaced it.

File: demo/helpers/demo-list-helper.html (hunk 2 above)
Comment (dotpointer) [reply demo/helpers/demo-list-helper.html]: Replaced it.

**Comment by dotpointer** [comment]
@Wurper Maybe unnecessary, but can you approve the review changes please? :)

**Pull request merged**